zio-schema-thrift/shared/src/main/scala/zio/schema/codec/ThriftCodec.scala

```diff
@@ -0,0 +1,1327 @@
+        case (StandardType.ZoneIdType, v: ZoneId) =>
+          p.writeString(v.getId)
+        case (StandardType.ZoneOffsetType, v: ZoneOffset) =>
+          p.writeI32(v.getTotalSeconds)
+        case (StandardType.Duration(_), v: Duration) =>
+          encodeRecord(None, durationStructure(), ListMap("seconds" -> v.getSeconds, "nanos" -> v.getNano))
+        case (StandardType.InstantType(formatter), v: Instant) =>
+          p.writeString(formatter.format(v))
+        case (StandardType.LocalDateType(formatter), v: LocalDate) =>
+          p.writeString(formatter.format(v))
+        case (StandardType.LocalTimeType(formatter), v: LocalTime) =>
+          p.writeString(formatter.format(v))
+        case (StandardType.LocalDateTimeType(formatter), v: LocalDateTime) =>
+          p.writeString(formatter.format(v))
+        case (StandardType.OffsetTimeType(formatter), v: OffsetTime) =>
+          p.writeString(formatter.format(v))
+        case (StandardType.OffsetDateTimeType(formatter), v: OffsetDateTime) =>
+          p.writeString(formatter.format(v))
+        case (StandardType.ZonedDateTimeType(formatter), v: ZonedDateTime) =>
+          p.writeString(formatter.format(v))
+        case (_, _) => ()
+      }
+
+    def writeFieldBegin(fieldNumber: Option[Short], ttype: Byte): Unit =
+      fieldNumber.foreach(
+        num =>
+          p.writeFieldBegin(
+            new TField("", ttype, num)
+          )
+      )
+
+    def encodePrimitive[A](fieldNumber: Option[Short], standardType: StandardType[A], value: A): Unit = {
+      writeFieldBegin(fieldNumber, getPrimitiveType(standardType))
+      writePrimitiveType(standardType, value)
+    }
+
+    def encodeSequence[A](fieldNumber: Option[Short], schema: Schema[A], v: Chunk[A]): Unit = {
+      writeFieldBegin(fieldNumber, TType.LIST)
+      p.writeListBegin(new org.apache.thrift.protocol.TList(getType(schema), v.size))
+      v.foreach(encodeValue(None, schema, _))
+    }
+
+    def encodeMap[K, V](fieldNumber: Option[Short], schema: Schema.MapSchema[K, V], v: Map[K, V]): Unit = {
+      writeFieldBegin(fieldNumber, TType.MAP)
+      p.writeMapBegin(new org.apache.thrift.protocol.TMap(getType(schema.ks), getType(schema.vs), v.size))
+      v.foreach {
+        case (k, v) =>
+          encodeValue(None, schema.ks, k)
+          encodeValue(None, schema.vs, v)
+      }
+    }
+
+    def encodeSet[A](fieldNumber: Option[Short], schema: Schema[A], v: Set[A]): Unit = {
+      writeFieldBegin(fieldNumber, TType.SET)
+      p.writeSetBegin(new org.apache.thrift.protocol.TSet(getType(schema), v.size))
+      v.foreach(encodeValue(None, schema, _))
+    }
+
+    def encodeOptional[A](fieldNumber: Option[Short], codec: Schema[A], v: Option[A]): Unit =
+      v.foreach(encodeValue(fieldNumber, codec, _))
+
+    def encodeValue[A](fieldNumber: Option[Short], schema: Schema[A], value: A): Unit =
```

Comment:
Can we add `//scalafmt: { maxColumn = 400, optIn.configStyleArguments = false }` for this method?

Reply:
:+1: